scripts/run_e2e_frontend.sh

```diff
@@ -2,16 +2,24 @@
 set -e
 
 
+echo ""
```

Comment:
Echo without params should always give you a newline.
```suggestion
echo
```

Reply:
done for all of those 👍